inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/convolution/convolution_kernel_b_fs_zyx_fsv16_imad.cpp

```diff
@@ -63,190 +63,210 @@ static size_t getOutBlock_X(const size_t output_size_x, const size_t stride_x, c
 
 namespace kernel_selector {
 
+inline size_t Convolution_kernel_b_fs_zyx_fsv16_imad::GetMaxThreadsPerDevice(const convolution_params& params) const {
+    constexpr size_t max_threads_per_compute_unit = 7;
+
+    return params.engineInfo.computeUnitsCount * max_threads_per_compute_unit;
```

Comment:
I think it makes sense to add `max_threads_per_compute_unit` field in `EngineInfo`